src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupApiRestClient.java

```diff
@@ -0,0 +1,253 @@
+        JsonElement result = gerritRestClient.getRequest(restPath);
+        return groupsParser.parseGroupInfo(result);
+    }
+
+    /** @return group name. */
+    public String name() throws RestApiException {
+        String restPath = getRequestPath() + "/name";
+        return gerritRestClient.getRequest(restPath).getAsString();
+    }
+
+    /**
+     * Set group name.
+     *
+     * @param name new name.
+     * @throws RestApiException
+     */
+    public void name(String name) throws RestApiException {
+        String restPath = getRequestPath() + "/name";
+        JsonElement result = gerritRestClient.putRequest(restPath, name);
+    }
+
+    /** @return owning group info. */
+    public GroupInfo owner() throws RestApiException {
+        String restPath = getRequestPath() + "/owner";
+        JsonElement result = gerritRestClient.getRequest(restPath);
+        return groupsParser.parseGroupInfo(result);
+    }
+
+    /**
+     * Set group owner.
+     *
+     * @param owner identifier of new group owner.
+     * @throws RestApiException
+     */
+    public void owner(String owner) throws RestApiException {
+        String restPath = getRequestPath() + "/owner";
+        JsonElement result = gerritRestClient.putRequest(restPath, owner);
+    }
+
+    /** @return group description. */
+    public String description() throws RestApiException {
+        String restPath = getRequestPath() + "/description";
+        return gerritRestClient.getRequest(restPath).getAsString();
+    }
+
+    /**
+     * Set group decsription.
+     *
+     * @param description new description.
+     * @throws RestApiException
+     */
+    public void description(String description) throws RestApiException {
+        String restPath = getRequestPath() + "/description";
+        JsonElement result = gerritRestClient.putRequest(restPath, description);
+    }
+
+    /** @return group options. */
+    public GroupOptionsInfo options() throws RestApiException {
+        throw new NotImplementedException();
+    }
+
+    /**
+     * Set group options.
+     *
+     * @param options new options.
+     * @throws RestApiException
+     */
+    public void options(GroupOptionsInfo options) throws RestApiException {
+        throw new NotImplementedException();
+    }
+
+    /**
+     * List group members, non-recursively.
+     *
+     * @return group members.
+     * @throws RestApiException
+     */
+    public List<AccountInfo> members() throws RestApiException {
+        String restPath = getRequestPath() + "/members";
+        JsonElement result = gerritRestClient.getRequest(restPath);
+        return groupsParser.parseGroupMembers(result);
+    }
+
+    /**
+     * List group members.
+     *
+     * @param recursive whether to recursively included groups.
+     * @return group members.
+     * @throws RestApiException
+     */
+    public List<AccountInfo> members(boolean recursive) throws RestApiException {
```

Comment:
I don't have any experience with mock frameworks, so I just tried to fix the code so it didn't ignore the method argument.  I wasn't sure about your preference for creating a new string or appending to a StringBuffer, so I didn't use the append approach.